### PR TITLE
Update styleguide input with InputButton

### PR DESCRIPTION
**Pull request by omninando**
#### What is the purpose of this pull request?
- Update styleguide input with InputButton

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/25524/alterar-componente-de-c%C3%B3digo-postal-do-address-form-de-input-button-para-inputbutton)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--checkoutio.myvtex.com/cart/add?&sku=298&qty=1&seller=1&sc=1)
2. Edit shipping to see new `InputButton`

#### Screenshots or example usage
n/a
#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by marcoskwkm** [comment]
Your test plan points to `master` workspace =P

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Changed
+
+- Styleguide input to show `InputButton` instead of `Input` with a `Button` suffix
```
Comment (marcoskwkm) [inline CHANGELOG.md]: nit: Period.

File: react/inputs/StyleguideInput/index.js [hunk 2]
```diff
@@ -64,45 +64,58 @@ class StyleguideInput extends Component {
...
+                })
+              }
+              onBlur={this.props.onBlur}
+              onChange={this.handleChange}
+              suffix={
+                <SpinnerLoading
+                  isLoading={!onSubmit && !submitLabel && loading}
+                />
+              }
+            />
```
Comment (marcoskwkm) [inline react/inputs/StyleguideInput/index.js]: suggestion: Move the common props to an object and use the spread operator to avoid the copy-pasta.

Comment (omninando) [reply react/inputs/StyleguideInput/index.js]: Done!

**Pull request merged**